Add README file

bboulesteix opened the pull request:
This pull request:

- Adds a README to the repository, in order to help marketers get started with the use and edition of the guide.
- Edits the 404 page of the site.
- Edits the content of the footer of the site, to cut one line and prevent another part from appearing on our 404 page.

bboulesteix committed:
Edit 404 page

This commit edits the content and the layout of our 404 page. It also removes useless CSS from the page.

bboulesteix committed:
Edit footer content

This commit edits the line about our license for brevity. It also prevents the paragraph about Memo Bank from appearing on our 404 page, where it won't be super helpful.

bboulesteix committed:
Add first draft of README.md file

This commit adds the first draft of our README file.

dirtyhenry commented on README.md:
```diff
@@ -0,0 +1,529 @@
+
+Memo Bank's growth guide is a financing guide aimed at established companies that are still growing fast (“scaling“ companies). The guide is maintained by [Memo Bank](https://memo.bank/en), a French bank built by entrepreneurs, for entrepreneurs who run small and medium businesses.
+
+This repository contains the code used to publish our guide at:[guide.memo.bank](https://guide.memo.bank). We use [Jekyll](https://jekyllrb.com) with the [Just the Docs](https://just-the-docs.github.io/just-the-docs/) theme. This site is built and served by [GitHub Page](https://pages.github.com).
```
There is a space missing:

```diff
+ This repository contains the code used to publish our guide at: [guide.memo.bank](https://guide.memo.bank).
- This repository contains the code used to publish our guide at:[guide.memo.bank](https://guide.memo.bank). 
```

bboulesteix replied:
Thanks @dirtyhenry. I'll fix it in a forthcoming commit. 👍

dirtyhenry commented on README.md:
```diff
@@ -0,0 +1,529 @@
+
+☝️ Note that “Terminal apps“ don't show up in your `Applications` folder. They're development tools, not real apps with a visual interface like Slack, iMovie, or Figma.
+
+#### Installing Ruby
```
macOS comes with a default Ruby version. Did you really need to go through these steps? 🤔

bboulesteix replied:
You're right about macOS, but Jekyll's documentation advises against using the default Ruby version that ships with macOS:

> To install Jekyll on macOS, you need a proper Ruby development environment. While macOS comes preinstalled with Ruby, we don’t recommend using that version to install Jekyll. This external article goes over the various reasons [why you shouldn’t use the system Ruby](https://www.moncefbelyamani.com/why-you-shouldn-t-use-the-system-ruby-to-install-gems-on-a-mac/).

— https://jekyllrb.com/docs/installation/macos/

dirtyhenry replied:
Interesting. Thanks for the link.

bboulesteix committed:
Fix typos and reorder sections

This commit fixes a few typos and reorders a few sections of the document to improve readability.
